<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EyePupilLocalizationClass</class>
 <widget class="QMainWindow" name="EyePupilLocalizationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EyePupilLocalization</string>
  </property>
  <property name="windowIcon">
   <iconset resource="eyepupillocalization.qrc">
    <normaloff>:/EyePupilLocalization/Resources/eye.ico</normaloff>:/EyePupilLocalization/Resources/eye.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>X绝对地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Y绝对地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>瞳孔半径</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdNumber_Rx"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_Ry"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_Rr"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Button_openvideo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开本地视频</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>右眼</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Reye">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X绝对地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y绝对地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>瞳孔半径</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdNumber_Lx"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_Ly"/>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_Lr"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>左眼</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Leye">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_x" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
    <widget class="QCustomPlot" name="customPlot_print" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>700</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="customPlot_y" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_print">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打印</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_opencamera">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="eyepupillocalization.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Button_openvideo</sender>
   <signal>clicked()</signal>
   <receiver>EyePupilLocalizationClass</receiver>
   <slot>on_pushButton_openvideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_print</sender>
   <signal>clicked()</signal>
   <receiver>EyePupilLocalizationClass</receiver>
   <slot>on_pushButton_print_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_opencamera</sender>
   <signal>clicked()</signal>
   <receiver>EyePupilLocalizationClass</receiver>
   <slot>on_pushButton_opencamera_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButton_openvideo_clicked()</slot>
  <slot>on_pushButton_print_clicked()</slot>
  <slot>on_pushButton_opencamera_clicked()</slot>
 </slots>
</ui>
